tsconfig.base.json

```diff
@@ -58,6 +58,7 @@
       "@fxa/shared/notifier": ["libs/shared/notifier/src/index.ts"],
       "@fxa/shared/otp": ["libs/shared/otp/src/index.ts"],
       "@fxa/shared/pem-jwk": ["libs/shared/pem-jwk/src/index.ts"],
+      "@fxa/shared/react/*": ["libs/shared/react/src/*"],
```

Comment:
Change required. Is this the alias that was generated by the nx generator? Why is it different from the typescript libraries?

Reply:
No, I updated it to match shared-assets. I'll revert.